documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -94,6 +94,11 @@ Capacity limits can be set for the following broker resources, using the standar
 
 Because Strimzi Kafka brokers are homogeneous, Cruise Control applies the same capacity limits to every broker it is monitoring.
 
+[NOTE]
+====
+Cruise Control assumes brokers resources to be homogeneous. Therefore, it is recommended to set value in `Kafka.spec.kafka.resources.requests` equal to `Kafka.spec.kafka.resources.limits`, so that Kubernetes will ensure broker pods have the same capacity and Guaranteed QoS, which also improves stability. This is especially important when using nodes with heterogeneous resources (CPU cores for example).
```

Comment:
```suggestion
We recommend that `Kafka.spec.kafka.resources.requests` and `Kafka.spec.kafka.resources.limits` are set to equal values. With these settings, broker pods will have the same capacity and quality of service. The settings can improve stability. This is especially important when using nodes with heterogeneous resources, such as CPU cores.
```

Reply:
Don't think we need the homogeneous line,as it's written in the sentence before the note.

Reply:
Perfect. Thanks.